docs/source/test-model.md

```diff
@@ -1,51 +1,56 @@
 # Test inference application deployed
 
-## Using notebook in JupyterHub
+If you want to test your application deployed in the cluster you can run the [integration test](https://github.com/thoth-station/elyra-aidevsecops-tutorial/tree/master/features) using [behave][1] package.
 
-If you want to test your application deployed in the cluster from JH image you can use the following notebook:
+Behave uses Behavior-driven development (or BDD), an agile software development technique that encourages and faciliate collaboration between developers, QA and non-technical or business participants.
 
-- Test model [Jupyter notebook](https://github.com/thoth-station/elyra-aidevsecops-tutorial/blob/master/notebooks/test_deployed_model.ipynb);
-
-(You need to have credentials (token) for the access to [Operate First][1] cluster and have access to the namespace where the application is deployed to run the above notebook).
-
-## From your local machine
-
-If you want to test the application created in this tutorial from your local machine:
-
-PRE-REQUISITE: Make sure you are logged in the cluster where the model is deployed.
-
-1. Install dependencies using [Pipenv](https://github.com/pypa/pipenv).
+1. Install [thamos][2] using pip:
 
 ```bash
-  pipenv install
+  pip install thamos
 ```
 
-or [micropipenv](https://pypi.org/project/micropipenv/):
+2. Create an env and install dependencies using dependencies using [thamos][2] from the root directory of the tutorial.
 
 ```bash
-  micropipenv install
+  python3 -m venv venv/ && . venv/bin/activate && thamos install -r test-model
 ```
 
-2. Start application.
+3. Run [behave][1] command from the root directory of the tutorial.
 
 ```bash
-  pipenv run python3 wsgi.py
+  DEPLOYED_MODEL_URL=<MODEL_DEPLOYED_URL> behave
```

Comment:
shall we put the op1st url in here, so that we got a working example?

Reply:
Yes sounds good!